src/main/java/seedu/todolist/model/task/Task.java

```diff
@@ -0,0 +1,70 @@
+package seedu.todolist.model.task;
+
+import seedu.todolist.commons.util.CollectionUtil;
+import seedu.todolist.model.tag.UniqueTagList;
+
+import java.util.Objects;
+
+/**
+ * Represents a Task in the address book.
+ * Guarantees: details are present and not null, field values are validated.
+ */
+public class Task implements ReadOnlyTask {
+
+    private Name name;
+
+    private UniqueTagList tags;
```

Comment:
Remnants from the address book code ? If you're not using tags for Tasks then you need to clean up any code related to `tags`.